jupyterlab/labbox_ephys_widgets_jp/src/extensions/extensionInterface.ts

```diff
@@ -119,6 +119,48 @@ export const sortingCurationReducer: Reducer<SortingCuration, SortingCurationAct
 }
 ////////////////////
 
+// Sorting selection
```

Comment:
Implementation of new selection/selectionDispatch -- similar structure as curation/curationDispatch